endaq/plot/plots.py

```diff
@@ -1229,3 +1274,173 @@ def pvss_on_4cp(
         height=size
     )
 
+
+def table_plot(
+        table: pd.DataFrame,
+        num_round: int = 2,
+        row_size: int = None,
+        font_color: str = None,
+        bg_color: str = None,
+        line_color: str = None
```

Comment:
Since `None` isn't an int or str, these should all use `typing.Optional[int]` or `typing.Optional[str]` as the type annotation... see earlier comment.